github/git_refs.go

```diff
@@ -50,6 +50,8 @@ type updateRefRequest struct {
 // GetRef fetches a single Reference object for a given Git ref.
 // If there is no exact match, GetRef will return an error.
 //
+// Note: the GitHub API can return multiple matches, if you wish to use this functionality please use the GetRefs() method.
```

Comment:
Also, "The" should be capitalized, since "Note:" is usually treated as a prefix to a complete sentence, rather than being considered a part of the sentence.

E.g., see https://github.com/golang/go/blob/f75b5e99ed19e97ea78e62a406d03c1632efa7d4/src/cmd/asm/internal/asm/parse.go#L795.